getting-started/bookshelf/bookshelf.go

```diff
@@ -80,8 +82,28 @@ func NewBookshelf(projectID string, db BookDatabase) (*Bookshelf, error) {
 		return nil, fmt.Errorf("storage.NewClient: %v", err)
 	}
 
+	loggingClient, err := logging.NewClient(ctx, projectID)
+	if err != nil {
+		log.Fatalf("logging.NewClient: %v", err)
+	}
+	logger := loggingClient.Logger("bookshelf-log")
+	noticeLogger := logger.StandardLogger(logging.Notice)
+	errLogger := logger.StandardLogger(logging.Error)
```

Comment:
I've been exploring structured logging, log correlation, and tracing lately for a microservices sample app on Cloud Run. It might be worth comparing notes.

Reply:
Simplified this based on our conversation. We can add more detail/complexity later if we decide it's worth it.